src/prettyprint.lua

```diff
@@ -0,0 +1,88 @@
+local COLORS = require("src.utils.consoleColor")
+local PASS_SYMBOL = COLORS.GREEN("√")
+local FAIL_SYMBOL = COLORS.RED("×")
+local PASS_HEADER = COLORS.WHITE_ON_GREEN_BOLD(" PASS ")
+local FAIL_HEADER = COLORS.WHITE_ON_RED_BOLD(" FAIL ")
+
+-- Matches Jest
+local TAB_SIZE = 2
+
+local NodeType = require("src.interface.testnodetype")
+local tablex = require("src.utils.tablex")
+
+local function getTabs(amount)
+	return (" "):rep(amount * TAB_SIZE)
+end
+
+--- Prints detailed reports about the test suites in the console.
+---@param results lest.TestSuiteResults
+local function printDetailedReports(results)
+	local testSuitesPassed = {}
+	for _, testSuite in pairs(results) do
+		testSuitesPassed[testSuite.name] = true
+
+		--- Recursively traverses a node.
+		---@param node lest.TestNode|lest.TestSuiteResults
```

Comment:
These types are invalid.

`lest.TestNode` is the test prior to being run.

Additionally, it is never being passed anything other than `lest.TestSuiteResults` because of the lack of recursion.